subsystem/cluster_test.go

```diff
@@ -5241,32 +5241,28 @@ var _ = Describe("Verify install-config manifest", func() {
 		networking, ok := installConfig["networking"].(map[interface{}]interface{})
 		Expect(ok).To(Equal(true))
 
-		// Networks are not included when fetching using V2GetClusterInstallConfig
-		// (the cluster is fetched without eager loading)
-		if networksIncluded {
-			// Validate 'clusterNetwork'
-			clusterNetwork, ok := networking["clusterNetwork"].([]interface{})
-			Expect(ok).To(Equal(true))
-			cidrEntry, ok := clusterNetwork[0].(map[interface{}]interface{})
-			Expect(ok).To(Equal(true))
-			cidr, ok := cidrEntry["cidr"].(string)
-			Expect(ok).To(Equal(true))
-			Expect(cidr).To(Equal(clusterCIDR))
-			// Validate 'machineNetwork'
-			machineNetwork, ok := networking["machineNetwork"].([]interface{})
-			Expect(ok).To(Equal(true))
-			cidrEntry, ok = machineNetwork[0].(map[interface{}]interface{})
-			Expect(ok).To(Equal(true))
-			cidr, ok = cidrEntry["cidr"].(string)
-			Expect(ok).To(Equal(true))
-			Expect(cidr).To(Equal(machineCIDR))
-			// Validate 'serviceNetwork'
-			serviceNetwork, ok := networking["serviceNetwork"].([]interface{})
-			Expect(ok).To(Equal(true))
-			cidr, ok = serviceNetwork[0].(string)
-			Expect(ok).To(Equal(true))
-			Expect(cidr).To(Equal(serviceCIDR))
-		}
+		// Validate 'clusterNetwork'
```

Comment:
@danielerez sure you have this info?
according to the comment the cluster is loaded from DB without this info

Reply:
Yeah, added 'UseEagerLoading' to getCluster call for that.